### PR TITLE
docs: wind request simbrief integration

### LINKED ISSUE #561: add wind requests feature
Possibly to be included in the beginner guide rather than a separate feature.

Planning to be part of the MCDU section as it's related and good practice to have the wind requests in when configuring the aircraft.

Reference PR:
- https://github.com/flybywiresim/a32nx/pull/7158

### PR DESCRIPTION
**Pull request by Valastiri**
closes #561 

## Summary
This PR is meant to takeover stale PR:

- #562 

Initial work courtesy of @USA-RedDragon including the feature. Please see the previous PR for more information.

### TODO

- [x] Add images of WIND REQUEST pages
- [x] Possibly include the wind req. within DIFSRIPP

### Location
- docs/fbw-a32nx/feature-guides/simbrief.md

Discord username (if different from GitHub): Valastiri#8902

### CODE REVIEW

File: docs/pilots-corner/beginner-guide/preparing-mcdu.md [hunk 1]
```diff
@@ -165,6 +165,19 @@ The cost index can be found in the image above.
 * Using the keypad type in `44`
 * Press LSK5L.
 
+!!! tip "Wind Requests"
+    Pilots can choose to import your wind data at this stage through the `WIND/TEMP` option by pressing LSK 4R. For more information on using this feature please see the guide below (*opens in new
+    tab*).
+
+    [Wind Request Guide](../../fbw-a32nx/feature-guides/simbrief.md#wind-request){target=new .md-button}
```
Comment (frankkopp) [inline docs/pilots-corner/beginner-guide/preparing-mcdu.md]: We usually don't open new windows for internal links, right?

```suggestion
!!! tip "Wind Requests"
    Pilots can choose to import your wind data at this stage through the `WIND/TEMP` option by pressing LSK 4R. For more information on using this feature please see the guide below.

    [Wind Request Guide](../../fbw-a32nx/feature-guides/simbrief.md#wind-request){.md-button}
```

**Review by frankkopp — APPROVED** [review]
Only one small comment - LGTM

File: docs/pilots-corner/beginner-guide/preparing-mcdu.md (hunk 1 above)
Comment (Valastiri) [reply docs/pilots-corner/beginner-guide/preparing-mcdu.md]: We don't, you are right slipped my mind. Thanks!

**Comment by Valastiri** [comment]
check build

**Pull request merged**